phenotrex/ml/trex_classifier.py

```diff
@@ -45,7 +54,7 @@ def __init__(self, random_state: int = None, verb: bool = False):
         self.random_state = np.random.RandomState(random_state)
         self.random_state_init = random_state
         self.verb = verb
-        self.vectorizer = CustomVectorizer(binary=True, dtype=np.bool, lowercase=False)
```

Comment:
Does that mean, `CustomVectorizer` is not used anymore at all? In this case, let's remove the class altogether.